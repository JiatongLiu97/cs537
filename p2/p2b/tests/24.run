./mysh tests/24.in
