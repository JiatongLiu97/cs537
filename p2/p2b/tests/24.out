tests/segfault
Going to segfault!
exit
